src/Case-TGV.f90

```diff
@@ -67,17 +68,24 @@ subroutine init_tgv (ux1,uy1,uz1,ep1,phi1)
              do i=1,xsize(1)
                 x=real(i-1,mytype)*dx
                 if(mhd_active) then
-                   ux1(i,j,k)= -two*sin(y)*cos(z)
-                   uy1(i,j,k)=  two*sin(x)*cos(z)
-                   uz1(i,j,k)=zero
                    !
-                   Bm(i,j,k,1)=-two*sin(y)
-                   Bm(i,j,k,2)= two*sin(two*x)
-                   Bm(i,j,k,3)=zero
+                   ! OTV
+                   ux1(i,j,k)=-two*sin(y)
+                   uy1(i,j,k)= two*sin(x)
+                   uz1(i,j,k)= zero
                    !
-                   Bmean(i,j,k,1)=zero
-                   Bmean(i,j,k,2)=zero
-                   Bmean(i,j,k,3)=zero
+                   if(nz<=2) then
```

Comment:
why "nz<=2"?

Reply:
I guess this is to distinguish between 3D and 2D cases.

Reply:
Yes initialisation is different between 2D and 3D. I guess this is OK and the only way to identify a 2D case.